Formula/socket_vmnet.rb

```diff
@@ -0,0 +1,46 @@
+class SocketVmnet < Formula
+  desc "Daemon to provide vmnet.framework support for rootless QEMU"
+  homepage "https://github.com/lima-vm/socket_vmnet"
+  url "https://github.com/lima-vm/socket_vmnet/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "da85362518c4ccfef3587240d380d52d7c7621c4567f1d9a476e1c17c5a7563b"
+  license "Apache-2.0"
+  head "https://github.com/lima-vm/socket_vmnet.git", branch: "master"
+
+  depends_on :macos
+
+  def install
+    # make: skip "install.launchd"
+    system "make", "install.bin", "install.doc", "VERSION=#{version}", "PREFIX=#{prefix}"
+    # Create optional launchd files
+    mkdir_p "#{prefix}/launchd"
+    Pathname.glob("launchd/*") do |file|
+      inreplace file, "/opt/socket_vmnet", opt_prefix
+      cp file, "#{prefix}/launchd/#{file.basename}"
+    end
```

Comment:
```suggestion
    # Create optional launchd files
    inreplace (buildpath/"launchd").children, "/opt/socket_vmnet", opt_prefix
    prefix.install "launchd"
```

Reply:
Updated to use `sudo brew services`, so no longer need to create launchd files here